core/sql/executor/HTableClient.java

```diff
@@ -1112,7 +1173,7 @@ public boolean release(boolean cleanJniObject) throws IOException {
               try {
                  future.get(30, TimeUnit.SECONDS);
               } catch(TimeoutException | InterruptedException e) {
-		  logger.error("Pre-fetch Thread is Cancelled, " + e);
+		  logger.error("Asynchronos Thread is Cancelled, " + e);
```

Comment:
Will fix it in my next checkin if there is no other critical issue to take care of